### ISSUE TITLE
Combining compute with callbaks

### ISSUE THREAD

**Issue by josejachuf** [gi0baro/weppy#151]
Given this model

```
class MyModel(Model)
    f1 = Field('integer')
    f2 = Field('integer')
    f3 = Field('integer')

    form_rw = {        
        'f3': False,
    }

    @compute('f3')
    def compute_f3(self, row):

        return row.f1 + row.f2

    @after_insert
    def my_function(self, fields, rid):     
        do_something(fields.f3) ########  f3 is not in fields
```

I have other ways to do what I want, but do not want to miss the comfort that gives this.

Jose

**Comment by gi0baro** [comment]
@josejachuf this is pyDAL related. Computed values (https://github.com/web2py/pydal/blob/3b918f08745b1a4877459e8bd750560d3c66db09/pydal/objects.py#L688) are not added to the fields passed to the callbacks (https://github.com/web2py/pydal/blob/3b918f08745b1a4877459e8bd750560d3c66db09/pydal/objects.py#L743).

Will open up a more structured issue on pyDAL regarding values in callbacks.

**Comment by josejachuf** [comment]
Thanks Giovanni

**Comment by gi0baro** [comment]
Implemented since 0ed86af
